.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```

Comment:
not opposed to it, but can I know the reason for removing this?

Reply:
sure.

> # And there's more
> ![image](https://private-user-images.githubusercontent.com/44438849/366264584-d6fec959-9206-4eef-8db2-dbc8a1269ef4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyMjUxNjEsIm5iZiI6MTcyNzIyNDg2MSwicGF0aCI6Ii80NDQzODg0OS8zNjYyNjQ1ODQtZDZmZWM5NTktOTIwNi00ZWVmLThkYjItZGJjOGExMjY5ZWY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDAwNDEwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2MDYxYjYzNzcxNWU0NjNmMzdmZTAzZjQ1MjMxZjc3Y2Y0YzIzODM3ZmEwOTQxNWQyYTFiYjI0NjdmMTFiYzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Zlgvguan8RacWz5fwZbhr4GDyluMgJUonmeYj_4xhEE)

This was another message that appeared when running husky. I originally posted this in the issue. I guess it is better to include it in the PR next time.